### ISSUE TITLE
Search doesn't work chinese

### ISSUE THREAD

**Issue by russellgao** [mkdocs/mkdocs#1824]
mkdocs 现在支持中文吗 ？

**Comment by waylan** [comment]
As explained in the [documentation][1], MkDocs uses [Lunr Languages][2] to offer multi-language search support. Unfortunately, as the docs state:

> Lunr Languages does not currently include support for Chinese or other Asian languages. However, some users have reported decent results using Japanese.

I just did a search, and it appears that there has been some work on adding Chinese support in MihaiValentin/lunr-languages#53. You should follow up there.

[1]: https://www.mkdocs.org/user-guide/configuration/#lang
[2]: https://github.com/MihaiValentin/lunr-languages#lunr-languages-----

**Comment by rapon233** [comment]
Noticing that this  lunr.zh.js project you mentioned is almost finished and has passed the checks, what should I do if I wanna make use of it?
Would there be any plan to merge into the stable release?

**Comment by waylan** [comment]
Before that gets included in MkDocs, it first needs to be committed to lunr-languages and then lunr-languages needs to release a new version which includes the support. You will have to ask the developers of lune-languages how long that will take. However, only after a new release of lune-languages will we update MkDocs to use the new version of lung-languages. Then, whenever we do our next release after that, it will be available for use. In other words, It may be some time.

**Comment by mouday** [comment]
i wish support  search for  chinese

**Comment by waylan** [comment]
Again, if you want to move this along, offer to help with MihaiValentin/lunr-languages#53. We cannot do anything until that is done. I have no knowledge or understanding of Chinese and am not able to help with that. So those of you who want support and have a knowledge need to go offer your assistance to lung-languages to get support added there.

**Comment by zhaohuaxishi** [comment]
Please reopen this issuse, sine the Chinese support has been merged into lunr-languages. https://github.com/MihaiValentin/lunr-languages/pull/53. I think if we use prebuid_index, we are now able to support chinese search.
